src/pkcs1v15.rs

```diff
@@ -436,6 +436,26 @@ where
     }
 }
 
+#[cfg(all(feature = "algoid", feature = "sha1"))]
+impl spki::DynAssociatedAlgorithmIdentifier for SigningKey<sha1::Sha1> {
+    fn algorithm_identifier(&self) -> spki::AlgorithmIdentifierOwned {
+        spki::AlgorithmIdentifierOwned {
+            oid: const_oid::ObjectIdentifier::new_unwrap("1.2.840.113549.1.1.5"),
+            parameters: None,
```

Comment:
All the parameters needs to be set explicitly to NULL as per https://www.rfc-editor.org/rfc/rfc4055#section-1.2

Reply:
(This is specific to x509 here, but I don't think it would hurt?)

Reply:
Done